staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/compilation_test.go

```diff
@@ -641,6 +641,63 @@ func TestCelCompilation(t *testing.T) {
 				invalidError("must evaluate to a bool"),
 			},
 		},
+		{
```

Comment:
Why no parse error, incorrect return type and type check error cases?

Reply:
Right, I've added compilation tests that check for parse errors/non-string return types/`string.format` issues.